### ISSUE TITLE
too few questions

### ISSUE THREAD

**Issue by binocarlos** [helixml/helix#14]
If you put a small bit of text like:

```
Bob lives at 6 Crow Terrace
```

It will generate a single question / answer pair and then axotl complains there are too few questions in the training data set

**Comment by lukemarsden** [comment]
yup we should catch this before it even gets to axolotl and encourage the user to give a minimum amount of training data

**Comment by lukemarsden** [comment]
relaetd to #58 (that's the "no data" case, this is the "too small" case, we should handle both with different, informative errors)
